test/CMakeLists.txt

```diff
@@ -55,6 +56,7 @@ list( APPEND test_output
   testoutput/hgodas_insitu.nc
   testoutput/hgodas_sst.nc
   testoutput/test_glider.nc
+  testoutput/ndbc_hfradar.nc
   testoutput/sfc_tv_obs_2018041500.nc4
```

Comment:
I believe you would also need to add ndbc_hfradar.nc to the /testoutput folder, like what you did for /testinput, so that it can be compared with the file generated in /testrun.